tests/test_unit.py

```diff
@@ -208,6 +209,41 @@ def test_client_pickle_unpickle(self):
             "original client and unpickled client must have the same state"
         )
 
+    @mock.patch('python_http_client.client.urllib')
+    def test_pickle_error(self, mock_lib):
```

Comment:
Is there a reason this test cannot go under the preexisting [@mock.patch(python_http_client.client.urllib)](https://github.com/sendgrid/python-http-client/blob/main/tests/test_unit.py#L150)?

Reply:
I put it in a separate function as it is a separate test so it is more clear what the issue is if the test fails. If you want I can move this test into the other test case that is already mocked.